tests/net/net_pkt/src/main.c

```diff
@@ -1152,8 +1152,6 @@ static void test_pkt_pull(void)
 static void test_net_pkt_append_memset(void)
 {
 	struct net_pkt *pkt;
-	struct net_buf *read_frag;
```

Comment:
This change is in wrong commit, should be in "net: pkt: Added new function net_pkt_append_memset() to prefill packet"